index.js

```diff
@@ -50,4 +62,19 @@ const noCommand = () => {
   console.log('Run: "blix help" to learn more about a command')
 }
 
+let currentNodeVersion = process.versions.node;
```

Comment:
We need node 8 to use async/await and thus must have a version check to make sure the user can use blix.